backend/src/main/java/com/agh/introwertycznelosie/data/Room.java

```diff
@@ -0,0 +1,146 @@
+package com.agh.introwertycznelosie.data;
+
+import org.springframework.data.domain.Range;
+
+import javax.persistence.*;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class Room {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private int recommendedCapacity;
+    private int maximalCapacity;
+    private String localization;
+    private int number;
```

Comment:
To number to jest numer sali, tak? To według mnie lepiej żeby to był String, bo np może być sala 3.24B

Reply:
46db096dad8496b77aa10568f84456387549050f